src/organizations/components/OrgProfileTab.tsx

```diff
@@ -43,105 +49,175 @@ const OrgProfileTab: FC = () => {
+              {label: 'Location', src: me.quartzMe?.regionName},
+            ].map(({label, src}) => {
+              if (!!src) {
+                return (
+                  <FlexBox
+                    key={`org-${label.toLowerCase()}`}
+                    direction={FlexDirection.Column}
+                    margin={ComponentSize.Large}
+                    alignItems={AlignItems.FlexStart}
+                  >
+                    <Heading
+                      element={HeadingElement.H4}
+                      weight={FontWeight.Regular}
+                      style={headingStyle}
+                    >
+                      {label}
+                    </Heading>
+                    <span style={{fontWeight: FontWeight.Light, width: '100%'}}>
+                      {src}
+                    </span>
+                  </FlexBox>
+                )
+              } else {
+                return null
+              }
+            })}
+          </FlexBox>
           <Heading
             element={HeadingElement.H4}
-            style={{marginTop: '16px', marginBottom: '4px'}}
+            style={headingStyle}
             weight={FontWeight.Regular}
           >
-            Organization ID
+            Cluster URL
           </Heading>
-          <div className="code-snippet" data-testid="code-snippet--orgid">
+          <div
+            className="code-snippet"
+            style={stretchStyle}
+            data-testid="code-snippet--userid"
+          >
             <div className="code-snippet--text">
               <pre>
-                <code>{org.id}</code>
+                <code>{me.quartzMe.clusterHost}</code>
               </pre>
             </div>
             <FlexBox
               className="code-snippet--footer"
               margin={ComponentSize.Medium}
+              stretchToFitWidth={true}
             >
               <CopyButton
-                text={org.id}
-                onCopy={generateCopyText('Organization ID', org.id)}
-                testID="copy-btn--orgid"
+                text={me.id}
+                testID="copy-btn--organizationUrl"
+                onCopy={generateCopyText(
+                  'Organization URL',
+                  me.quartzMe.clusterHost
+                )}
               />
-              <label
-                className="code-snippet--label"
-                data-testid="org-profile--name"
-              >
-                {`${org.name} |`} <b> Organization ID </b>
-              </label>
             </FlexBox>
           </div>
         </FlexBox.Child>
+
+        <FlexBox.Child
+          style={{...stretchStyle, ...sectionStyle}}
+          testID="common-ids--panel"
+        >
+          <h4>Common IDs</h4>
+          {[
+            {id: 'userid', humanId: 'User ID', src: me.id, label: me.name},
+            {
+              id: 'orgid',
+              humanId: 'Organization ID',
+              src: org.id,
+              label: org.name,
+            },
```

Comment:
nit: shouldn't the `humanId` be the `label` and the `label` be a `name`?

Reply:
done